advisories/github-reviewed/2025/01/GHSA-58fx-7v9q-3g56/GHSA-58fx-7v9q-3g56.json

```
@@ -1,13 +1,13 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-58fx-7v9q-3g56",
-  "modified": "2025-02-12T18:32:43Z",
+  "modified": "2025-01-28T20:40:01Z",
   "published": "2025-01-28T18:31:28Z",
   "aliases": [
     "CVE-2024-13484"
   ],
   "summary": "ArgoCD Namespace Isolation Break",
```

```suggestion
  "summary": "Red Hat OpenShift GitOps Namespace Isolation Break",
```
Forgot to suggest title change.